ionics_fits/models/laser_rabi.py

```diff
@@ -199,6 +209,8 @@ def estimate_parameters(
                 model_parameters["n_bar"].heuristic = 1
             if "alpha" in model_parameters.keys():
                 model_parameters["alpha"].heuristic = 1
+            if "zeta" in model_parameters.keys():
```

Comment:
This is getting a bit messy with all these special cases. We should consider putting in place an automated way of handling this.

Reply:
(e.g. have the model take a heuristic function for the distribution parameters)

Reply:
Agreed, but I'd prefer not to change it in this PR and instead introduce a new one for this purpose at some point in the future.